src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -203,6 +203,27 @@ with DefaultTimeout with ImplicitSender with WordSpecLike with Matchers with Bef
     (wma, workflowId)
   }
 
+  def runWdlAndAssertOutputs2(sampleWdl: SampleWdl,
```

Comment:
this is my fault.  I was fixing the rebase and I must have accidentally left this in